DependencyInjection/Configuration.php

```diff
@@ -42,6 +42,12 @@ public function getConfigTreeBuilder()
                 ->end()
                 ->scalarNode('token_ttl')
                     ->defaultValue(3600)
+                    ->validate()
+                        ->ifTrue(function ($ttl) {
+                            return !is_numeric($ttl);
```

Comment:
this check has been moved at runtime in order to allow use of `%env()%` parameters in https://github.com/lexik/LexikJWTAuthenticationBundle/pull/325. The point of using `is_numeric()` is to support env vars too.
About allowing null, this has been changed accidentally and rediscussed in https://github.com/lexik/LexikJWTAuthenticationBundle/issues/250, having tokens with an infinite lifetime is not something we want to support regarding security